app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id'
```

Comment:
- [ ] Optional you do not need a class_name 'Post' nor the foriegn_key.
- [ ]  It will be nice to have Rails active callbacks added.